src/components/AddPersonForm/index.jsx

```diff
@@ -122,78 +101,86 @@ class AddPersonForm extends React.Component {
     console.log('currentlyEditingPerson', currentlyEditingPerson)
     return (
       <React.Fragment>
-        <Form onSubmit={this.handleSubmit} {...formItemLayout} className="add-person-form" >
-          <h1>{formTitle}</h1>
-          <h4>Person info</h4>
-
-          <Form.Item
-            {...noLabelFormItemLayout}
-          >
-            {getFieldDecorator('displayName', {
-              rules: [{ required: true }],
-            })(
-              <Input placeholder="Full Name" />
-            )}
-          </Form.Item>
-          <Form.Item
-            {...noLabelFormItemLayout}
-          >
-            {getFieldDecorator('party', {
-              rules: [{ required: true, }],
-            })(
-              <Select onChange={this.handleChange}>
-                <Option value="Democratic">Democratic</Option>
-                <Option value="Republican">Republican</Option>
-                <Option value="Independent">Independent</Option>
-
-              </Select>
-            )}
-          </Form.Item>
-          <Form.Item
-            {...noLabelFormItemLayout}
-          >
-            {getFieldDecorator('in_office', {
-              valuePropName: 'checked',
-              initialValue: false,
-            })(
-              <Checkbox onChange={this.handleChange}>
-                Currently in office
-              </Checkbox>
-            )}
-          </Form.Item>
-          <Form.Item
-            {...noLabelFormItemLayout}
-          >
-            {getFieldDecorator('is_candidate', {
-              valuePropName: 'checked',
-              initialValue: false,
-            })(
-              <Checkbox onChange={this.handleChange}>
-                Is running for an office
-              </Checkbox>
-            )}
-          </Form.Item>
-
-          <Form.Item  {...noLabelFormItemLayout}>
-            <Button disabled={!this.state.canSubmit} type="primary" htmlType="submit" className="login-form-button">
-              Save to database
-            </Button>
-          </Form.Item>
-        </Form>
-        <h4>Role info</h4>
-        {getFieldValue('in_office') && currentlyEditingPerson &&
+        {this.state.allFormsComplete && <h2>New member successfully added to the database</h2>}
```

Comment:
small thing but should be 'new person' instead of 'new member' since it can be a candidate

Reply:
fixed